tasks/configure.yml

```diff
@@ -1,5 +1,17 @@
 ---
 
+- name: Add thinpool configuration
+  become: yes
+  template:
+    dest: /etc/docker/daemon.json
+    group: root
+    mode: 0444
+    owner: root
+    src: daemon.json.j2
+  when: docker.thin_pool_device is not none and ansible_distribution_release != "trusty"
+  notify:
+    - restart docker
+
 - name: Ensure Docker service is restarted
```

Comment:
Mega favour here but can you change this to 'Ensure Docker service is started'? Mistake from my initial PR :)

Reply:
copy pasta